Sources/TuistCore/Models/Scheme.swift

```diff
@@ -153,6 +153,26 @@ public class TestAction: Equatable {
     }
 }
 
+public struct TestableTarget: Equatable, Hashable {
```

Comment:
Similar comment here in regards to defining the conformance of `ExpressibleByStringLiteral` here.